### PR TITLE
[PRTL-3200 & PRTL-3273] scrnaseq: beta chip & endpoint integration

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket Number

PRTL-3200

## Environment to be used in testing

- [ ] `prod`
- [ ] `dev-oicr`
- [x] `qa-yellow`

### CODE REVIEW

File: src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.js [hunk 1]
```diff
@@ -85,9 +82,28 @@ const SelectScrnaSeq = ({
         >
         <Icon />
         <Column style={{ flex: 1 }}>
-          <h1 style={{ fontSize: '2rem' }}>{label}</h1>
+          <h1
```
Comment (samrichca) [inline src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.js:85]: actual change

File: src/packages/@ncigdc/components/analysis/AnalysisResult.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (caravinci) [inline src/packages/@ncigdc/components/analysis/AnalysisResult.js:1]:
```suggestion
```

File: src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.js [hunk 3]
```diff
@@ -85,9 +82,28 @@ const SelectScrnaSeq = ({
...
+                marginLeft: '1rem',
+              }}
+              />
+
+          </h1>
           <p>Try out the beta release of our new analysis tool for single cell RNA sequencing data.  Display a demo of different clustering visualizations for this data from a sample case.</p>
-          <p>Please send us your feedback at: <a href="mailto:support@nci-gdc.datacommons.io">support@nci-gdc.datacommons.io</a></p>
+          <p>
+            Please send us your feedback at:
+            {' '}
```
Comment (caravinci) [inline src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.js:104]:
```suggestion
            {'Please send us your feedback at: '}

```

File: src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.relay.js [hunk 4]
```diff
@@ -52,11 +52,10 @@ export default (Component) =>
   )((props) => {
     return (
       <Query
-        parentProps={props}
-        name="SelectScrnaSeqCases"
-        minHeight={387}
-        variables={variables}
         Component={Component}
+        minHeight={387}
```
Comment (caravinci) [inline src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.relay.js:56]: What's this number for/from?

File: src/packages/@ncigdc/modern_components/SCRNASeq/AnalysisDownloadsButton/AnalysisDownloadsButton.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/AnalysisDownloadsButton/AnalysisDownloadsButton.js:1]:
```suggestion
```

Comment (caravinci) [reply src/packages/@ncigdc/modern_components/SCRNASeq/AnalysisDownloadsButton/AnalysisDownloadsButton.js:1]: Unnecessary

File: src/packages/@ncigdc/modern_components/SCRNASeq/SCRNASeq.relay.js [hunk 6]
```diff
@@ -57,12 +59,17 @@ export default (Component: ReactClass<*>) =>
...
       // },
       getWholeTsv: ({
+        analysisInfo: { file_id },
         setData,
         setLoading,
       }) => async () => {
         // const file = await import('./stubData/seurat.tsv');
         // const { body: stream } = await fetch(`http://localhost:3000${file}`);
-        const { body: stream } = await fetch('https://gist.githubusercontent.com/caravinci/e538391f6681348446af21127d30e4e7/raw/e3c8569f3b8fa50a00823486ccd7a0e3c723bee1/seurat.tsv');
+        // const { body: stream } = await fetch('https://gist.githubusercontent.com/caravinci/e538391f6681348446af21127d30e4e7/raw/e3c8569f3b8fa50a00823486ccd7a0e3c723bee1/seurat.tsv');
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/SCRNASeq.relay.js:68]:
```suggestion
```

**Review by caravinci — CHANGES_REQUESTED** [review]

File: src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.relay.js (hunk 4 above)
Comment (samrichca) [reply src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.relay.js:56]: it was originally the default height for card plots in clinical analysis, but it works ok here. it sets the height of the element before the query loads

**Review by caravinci — APPROVED** [review]
Squash away! 👯

**Pull request merged**